source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -316,11 +308,9 @@ package:
 
        - `Document <{+api+}/apidocs/bson/org/bson/Document.html>`__
        - `BsonDocument <{+api+}/apidocs/bson/org/bson/BsonDocument.html>`__
-       - ``Map<String, Object>``
-
-       .. seealso::
-
-          :ref:`BsonExtraElements Annotation Example <bsonextraelements-annotation-code-example>`
+       - ``Map<String, Object>``  
+       
+       :ref:`BsonExtraElements Annotation Example <bsonextraelements-annotation-code-example>`
```

Comment:
```suggestion
       For more information on using the annotation, see the below :ref:`BsonExtraElements Annotation Example <bsonextraelements-annotation-code-example>` on this page.
```
Suggestion: Same as above, think it would be nice to introduce ex. in a sentence.